CliClient/locales/it_IT.po

```diff
@@ -566,6 +568,13 @@ msgid ""
 "\n"
 "For example, to create a notebook press `mb`; to create a note press `mn`."
 msgstr ""
+"Benvenuto in Joplin!\n"
```

Comment:
Why do you have several sentences instead of just the one above?

Also, I think there's a space missing on line 576 between digita and `mb`.

Reply:
Sorry, my fault, I didn't see the entire file. All good.